modules/ROOT/pages/dw-core-functions-daysbetween.adoc

```diff
@@ -3,29 +3,44 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<daysbetween1>>
-
 
 [[daysbetween1]]
 == daysBetween(Date, Date): Number
 
 Returns the number of days between two dates.
```

Comment:
So not inclusive of the specified dates? in the example it seems to include either the from or the to date?